demo/demo_primary.py

```diff
@@ -478,17 +421,14 @@ def enforce_jail(fname, expected_containing_dir):
 
 def get_image_for_ecu(ecu_serial):
   """
-  Behaves differently for partial-verification Secondaries communicating across
-  CAN and full-verification Secondaries communicating via xmlrpc.
-
-  For PV Secondaries on CAN:
-    Employs C-based libuptane library to send the image binary data to the
-    CAN ID noted in configuration as matching the given ECU Serial.
-
-  For FV Secondaries via XMLRPC:
-    Returns:
-     - filename of the image, relative to the targets directory
-     - binary image data in xmlrpc.Binary format
+  Intended to be called via XMLRPC by the Secondary client, either partial or
+  full verification.
+
+  Returns the following to the requesting Secondary:
+     - filename of the firmware image assigned it by the Director and validated
```

Comment:
-it?

Reply:
"it" here is the requesting Secondary.

Returns to X the filename assigned it by Y